packages/webapp/pages/squads/create.tsx

```diff
@@ -123,7 +123,7 @@ function CreatePost(): ReactElement {
         <TabContainer<WriteFormTab>
           onActiveChange={(active) => setDisplay(active)}
           controlledActive={display}
-          shouldMountInactive
+          shouldMountInactive={false}
```

Comment:
If we continue mounting inactive tabs, there will be a conflict with the ids. IIRC, the mounting was to keep the state of the content when switching tabs, but since we use local cache, they are retained. With that, no issue in turning this into false.